spec/models/move_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Move, type: :model do
-  # test goes here
+  it "should determine that player white is up" do
+    FactoryGirl.create(:active_game)
+    move1 = Move.create(game_id: game.id)
+    move2 = Move.create(game_id: game.id)
+    move3 = Move.create(game_id: game.id)
+    expect(move3.turn).to eq 111
+  end
+
+  it "should determine that player black is up" do
+    game = Game.create(active: true, user_id_black: 110, user_id_white: 111, index_moves_on_game_id: )
```

Comment:
It should be. 
But the test as written anyway doesn't work. Those moves don't populate to the game - game.moves.count on that comes back 0. I can't figure out how to write the test.